api/V1/recordingUtil.ts

```diff
@@ -246,21 +250,25 @@ async function report(request) {
       audioBaitDelta,
       audioBaitVolume,
       urljoin(recording_url_base, r.id.toString()),
-      cacophonyIndex
+      cacophonyIndex,
+      speciesClassifications
     ]);
   }
   return out;
 }
 
-function getCacophonyIndex(r) {
-  let cacophonyIndex = null;
-  if (r.additionalMetadata && r.additionalMetadata.analysis) {
-    cacophonyIndex = r.additionalMetadata.analysis.cacophony_index;
-    if (cacophonyIndex) {
-      cacophonyIndex = cacophonyIndex.map(val => val.index_percent).join(";");
-    }
-  }
-  return cacophonyIndex;
+function getCacophonyIndex(recording: Recording): string | null {
+  return (recording.additionalMetadata as AudioRecordingMetadata)?.analysis?.cacophony_index
+    ?.map(val => val.index_percent)
+    .join(";");
+}
+
+function getSpeciesIdentification(recording: Recording): string | null {
+  return (recording.additionalMetadata as AudioRecordingMetadata)?.analysis?.species_identify
+    ?.map(
+      classification => `${classification.species}, ${classification.begin_s}`
```

Comment:
would this be better being a ':' instead of a comma to sepearte?

Reply:
Done